tutor/src/components/task-plan/homework/loading-exercises-mixin.cjsx

```diff
@@ -12,7 +13,8 @@ LoadingExercisesMixin =
   componentWillUnmount: -> ExerciseStore.removeChangeListener(@exerciseLoadListenerOnUpdate)
 
   requestExerciseLoad: ->
-    ExerciseActions.loadForCourse(@props.courseId, @props.sectionIds, @props.ecosystemId)
+    ecosystemId = @props.ecosystemId or TaskPlanStore.getEcosystemId(@props.planId, @props.courseId)
```

Comment:
No if you look at the api action https://github.com/openstax/tutor-js/blob/master/tutor/src/api/index.coffee#L92 it only uses the `ecosytemId` if it's present.